doc/script_commands.txt

```diff
@@ -1904,57 +1904,61 @@ will result in error and termination of the script.
 ---------------------------------------
 
 *function <function name>;
-*<function name>{(<argument>, ...<argument>)};
```

Comment:
something here bothers me: we're using curly braces to denote optional arguments, but in this case literal curly braces are required; how should this be written? escaped curly braces (`\{ <code> \}`) or a custom sequence like we do for `%TAB%`?